components/web/apache2-modules/mod_auth_mellon/Makefile

```diff
@@ -19,24 +19,25 @@
 # CDDL HEADER END
 #
 # Copyright (c) 2011, 2015, Oracle and/or its affiliates. All rights reserved.
+# Copyright (c) 2022 Niklas Poslovski
 #
 
 BUILD_BITS=		64
 
```

Comment:
Have you tried to build this with openssl-1.1?
Add
USE_OPENSSL11= yes

and rerun "gmake REQUIRED_PACKAGES" to find out. Some package need more special settings, some won't work with 1.1 at all, though.